locales/en.default.schema.json

```diff
@@ -699,27 +691,33 @@
           "default": "Blog posts",
           "label": "Heading"
         },
+        "layout_header": {
+        "content": "Layout"
+        },
+        "text_header": {
+        "content": "Text"
+        },        
         "blog": {
           "label": "Blog"
         },
         "post_limit": {
-          "label": "Number of blog posts to show"
-        },
+          "label": "Post count"
+        }, 
         "columns_desktop": {
-          "label": "Number of columns on desktop"
+          "label": "Columns"
         },
         "show_view_all": {
-          "label": "Enable \"View all\" button if blog includes more blog posts than shown"
+          "label": "\"View all\" button",
+          "info": "Visible if blog has more posts than shown"
```

Comment:
Not too sure myself when it should end with a period or not.

Reply:
The new guidance is no period unless it's more than once sentence. So most of these won't have a period unless they're followed by a "Learn more" link.